### PR TITLE
Added intro page

### PR DESCRIPTION
**Pull request by loievskyi**

### CODE REVIEW

File: src/app/booking/Intro.js [hunk 1]
```diff
@@ -0,0 +1,95 @@
...
+      <Typography
+        mt={5}
+        color="text.secondary"
+        variant="introSubheading"
+        display={{ xs: "none", md: "block" }}
+      >
+        We know when kids are going to be Out Of School so why not plan by
+        calendar instead by activities. It’s that simple, easy and intuitive.
+      </Typography>
+      <Typography
```
Comment (shuryhin-oleksandr) [inline src/app/booking/Intro.js:33]: Should be 19 px at desktop

File: src/app/booking/Intro.js [hunk 2]
```diff
@@ -0,0 +1,95 @@
...
+  const STEPS = [
+    "Select activities",
+    "Activities are added to your child’s calendar",
+    "Pay for all bookings in one go",
+  ];
+
+  return (
+    <Box textAlign="center" {...props}>
+      <Typography my={3} variant="h1">
+        Discover, book and track activities without clicking,
```
Comment (shuryhin-oleksandr) [inline src/app/booking/Intro.js:21]: Orange text above should be here.

File: src/app/booking/Intro.js [hunk 3]
```diff
@@ -0,0 +1,95 @@
...
+        typing and scrolling marathons
+      </Typography>
+      <Typography
+        mt={5}
+        color="text.secondary"
+        variant="introSubheading"
+        display={{ xs: "none", md: "block" }}
```
Comment (shuryhin-oleksandr) [inline src/app/booking/Intro.js:28]: Use {mdUp && syntax here.

File: src/components/ThemeRegistry/theme.js [hunk 4]
```diff
@@ -67,6 +72,13 @@ const theme = createTheme({
       fontWeight: 700,
       color: theme.palette.orange.main,
     }),
+    introSubheading: {
```
Comment (shuryhin-oleksandr) [inline src/components/ThemeRegistry/theme.js:75]: font naming?

File: src/app/booking/Intro.js (hunk 1 above)
Comment (loievskyi) [reply src/app/booking/Intro.js:33]: Why?
image

File: src/app/booking/Intro.js (hunk 2 above)
Comment (loievskyi) [reply src/app/booking/Intro.js:21]: Ok, I'll add it. This inscription was added after I created the Intro (screenshot from slack from figma)

image

**Pull request closed without merging**